frontend/src/metabase/public/components/EmbedModal/StaticEmbedSetupPane/tests/common.unit.spec.tsx

```diff
@@ -477,7 +475,7 @@ describe("Static Embed Setup phase", () => {
         ).toBeVisible();
 
         const link = within(
-          screen.getByLabelText("Playing with appearance options"),
+          screen.getByLabelText("Customizing look and feel"),
```

Comment:
Since the removal of the second section, all the settings belong to the first section now.